src/test/scala/nl.knaw.dans.easy.bag2deposit/ddm/ProvenanceSpec.scala

```diff
@@ -152,8 +152,8 @@ class ProvenanceSpec extends AnyFlatSpec with FileSystemSupport with XmlSupport
     normalized(xml) shouldBe normalized(
       <prov:provenance xsi:schemaLocation={provLocations} xmlns:dct="http://purl.org/dc/terms/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:prov="http://easy.dans.knaw.nl/schemas/bag/metadata/prov/" xmlns:ddm="http://easy.dans.knaw.nl/schemas/md/ddm/">
         <prov:migration app="EasyConvertBagToDepositApp" version="1.0.5" date="2020-02-02">
```

Comment:
like the other `prov:file` elements, with a new element type `prov:encoding`. And I added an attribute `filename` to the `file` element to indicate the source file
`<prov:file filename="dataset.xml">     `          
    `<prov:old><prov:encoding><![CDATA[<E2><80><93> <E2><80><98> <e2><80><99> <cf><be>]]></prov:encoding></prov:old>`
    `<prov:new><prov:encoding>– ‘ ’ Ͼ</prov:encoding></prov:new>`
` </prov:file>`